test/scenarios.py

```diff
@@ -2171,6 +2171,392 @@
             ),
         },
     ),
+    Scenario(
+        "scenario11-2",
+        "Initate a buch of transfer and stop the sender. Expect immediate cancelation",
```

Comment:
Maybe we could use `action.Repeated` for creating N transfers with the same file, then awaiting for at least several of them to start(now its done using count=4) and then stop? This would reduce the testcode and would make it more obvious what is being tested.

I would also add this to the description that we're deliberately waiting for some
transfers to start so it would be more obvious instead of scanning through the scenario code :)

Reply:
done